CHANGES.md

```diff
@@ -157,6 +157,11 @@ OpenSSL 3.0
 
    *David von Oheimb*
 
+ * BIO_do_connect and BIO_do_handshake have been extended:
+   if domain name resolution yields multiple IP addresses all of them are tried.
```

Comment:
Isn't if -> If more correct gramatically?
Also I would like to see something like: "If domain name resolution yields multiple IP addresses all of them are tried after connect() failures." or "until some succeeds." or "before returning an error."

Reply:
> Isn't if -> If more correct grammatically?

It turns out that it depends on BE vs. AE and likely has become a matter of taste:
https://www.grammarly.com/blog/capitalization-after-colons/

BTW, which is the preferred flavor of English in OpenSSL these days? 
Looks like it used to be British, but in general American seems to dominate also the IT world.

Reply:
> Also I would like to see something like: "If domain name resolution yields multiple IP addresses all of them are tried after connect() failures."

Done, also for the other two occurrences.

Reply:
@t8m, is this fine now?